src/client/datascience/export/exportManagerFileOpener.ts

```diff
@@ -22,24 +24,35 @@ export class ExportManagerFileOpener implements IExportManager {
     ) {}
 
     public async export(format: ExportFormat, model: INotebookModel): Promise<Uri | undefined> {
-        const reporter = this.progressReporter.createProgressIndicator(`Exporting to ${format}`); // need to localize
+        const reporter = this.progressReporter.createProgressIndicator(`Exporting to ${format}`);
         let uri: Uri | undefined;
         try {
             uri = await this.manager.export(format, model);
         } catch (e) {
             await this.showExportFailed(e);
```

Comment:
Please log the error info.
```suggestion
			traceError('Export failed', e);
            await this.showExportFailed(e);
```